src/features/auth/pages/login/Login.web.test.tsx

```diff
@@ -36,26 +36,30 @@ describe('<Login/>', () => {
     })
   })
 
-  it('should display forced login help message when the query param is given', () => {
+  it('should display forced login help message when the query param is given', async () => {
     useRoute.mockReturnValueOnce({ params: { displayForcedLoginHelpMessage: true } })
-    const { queryByRole } = renderLogin()
+    renderLogin()
 
-    const snackBar = queryByRole('status')
+    const snackBar = screen.queryByRole('status')
 
-    expect(snackBar).toHaveTextContent(
-      'Pour sécuriser ton pass Culture, tu dois confirmer tes identifiants tous les 30 jours.'
-    )
+    await waitFor(() => {
+      expect(snackBar).toHaveTextContent(
+        'Pour sécuriser ton pass Culture, tu dois confirmer tes identifiants tous les 30 jours.'
+      )
+    })
```

Comment:
Si t'utilises `findBy` au lieu de `queryBy` :
```suggestion
      expect(snackBar).toHaveTextContent(
        'Pour sécuriser ton pass Culture, tu dois confirmer tes identifiants tous les 30 jours.'
      )
```